service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/StructuredRecordMappingService.java

```diff
@@ -136,11 +136,8 @@ public String mapStructuredRecordToEhrExtractXml(GetGpcStructuredTaskDefinition
         return outputMessageWrapperMapper.map(structuredTaskDefinition, ehrExtractContent);
     }
 
-    public String buildSkeletonEhrExtractXml(
-        GetGpcStructuredTaskDefinition structuredTaskDefinition, Bundle structuredRecord, String documentId
+    public String buildSkeletonEhrExtractXml(String realEhrExtract, String documentId
```

Comment:
I'm not sure how I feel about this indirection. Could the `ehrExtractMapper` method be called directly instead?